packages/edit-site/src/components/canvas-spinner/style.scss

```diff
@@ -4,4 +4,8 @@
 	display: flex;
 	align-items: center;
 	justify-content: center;
+
+	circle {
+		stroke: rgba($black, 0.5);
```

Comment:
I'd change this to opacity 0.3.

Reply:
👍 Done in 6222730ffee667c2543c241b88aa20acbeea9999